frontend/src/components/Body/WholeBody/WholeBody.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import BodyOne from '../BodyOne/BodyOne';
+import BodyTwo from '../BodyTwo/BodyTwo';
+import BodyThree from '../BodyThree/BodyThree';
+import '../BodyOne/BodyOne.css';
+import '../BodyTwo/BodyTwo.css';
+import '../BodyThree/BodyThree.css';
+
+function Body() {
+    return (
+        <div>
+            <BodyOne />
+            <BodyTwo />
+            <BodyThree />
```

Comment:
on naming in general, one of the important concepts in coding is the idea of [self documenting code](https://swimm.io/learn/documentation-tools/tips-for-creating-self-documenting-code#:~:text=code%20even%20clearer-,What%20is%20self%2Ddocumenting%20code%3F,the%20need%20for%20extensive%20documentation.) which is somewhat already descriptive, but in practice we want to reduce the mental overhead as much as possible for someone new (or even yourself, just further down the line) to understand your code. While it may make sense right now contextually to name these components BodyOne and such, this is actually not terribly descriptive in and of itself. It would be easier to name something like `Testimonials` or `ProductFeatures` or etc based on the content and usage of the component.